pom.xml

```diff
@@ -17,7 +17,8 @@
 		<spring.cloud.gcp.version>1.2.8.RELEASE</spring.cloud.gcp.version>
 		<common.version>10.49.4</common.version>
 		<java.version>17</java.version>
-		<surefire.version>2.20</surefire.version>
+		<surefire.version>3.2.2</surefire.version>
+		<skipSurefire>false</skipSurefire>
```

Comment:
Adding this to make it quicker to run maven builds when investigating the failsafe integration tests during development